migrate/0004_add_game_pinning.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    add_column :games, :pin_version, String, null: true, default: nil
```

Comment:
i don't think we need this, we can add it into settings

Reply:
fixed :)